src/doc/src/reference/workspaces.md

```diff
@@ -33,21 +33,24 @@ In the `Cargo.toml`, the `[workspace]` table supports the following sections:
 ## The `[workspace]` section
 
 To create a workspace, you add the `[workspace]` table to a `Cargo.toml`:
+
 ```toml
 [workspace]
 # ...
 ```
 
+The directory where the workspace's Cargo.toml is located then becomes so-called the workspace root.
+
 At minimum, a workspace has to have a member, either with a root package or as
 a virtual manifest.
```

Comment:
This is splitting up the text.  One way to read this is we are describing the minimum of what is needed in a workspace
1. A `[workspace]` table
2. A member

and this inserts the workspace root definition inbetween those

Reply:
If it doesn't harmonize, it's still the previous one that makes more sense and flows better.